packages/@glimmer/test-helpers/lib/environment.ts

```diff
@@ -1086,34 +1086,6 @@ function populateBlocks(blocks: BlockMacros, inlines: InlineMacros): { blocks: B
     builder.evaluate(sexp[5]);
   });
 
-  blocks.add('-with-dynamic-vars', (sexp, builder) => {
```

Comment:
Both of these were removed from the environment, but for some reason implemented just in the test environment.